### ISSUE TITLE
pylint reports maybe-no-member for calls to min/max on numpy array

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#366]
Originally reported by: **Luke Lee (BitBucket: [durden](http://bitbucket.org/durden), GitHub: @durden?)**

---

Pylint is attempting to detect some numpy array usage but doesn't seem to 'guess' correctly when using the min/max methods.  Is this something that can be fixed or is it a known issue due to numpy/pylint/python dynamic typing?

Here's an example that I've seen pylint complain about:

``` python
import numpy

array = numpy.array([1, 2, 1, 3])
unique = numpy.unique(array)
print unique.max()
```

I see this type of warning:

E:155,26: Instance of 'tuple' has no 'max' member (but some types could not be inferred) (maybe-no-member)

I'm using pylint 1.3.1 and astroid 1.2.1 with numpy 1.8.0

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/366

**Comment by PCManticore** [comment]
This is fixed in master.
